tests/Functional/TestKernel.php

```diff
@@ -64,6 +65,15 @@ public function registerBundles(): iterable
         yield new PurgatoryBundle();
     }
 
+    public function shutdown(): void
+    {
+        $handler = set_exception_handler('var_dump');
+        restore_exception_handler();
+        if (\is_array($handler) && $handler[0] instanceof ErrorHandler) {
+            restore_exception_handler();
+        }
+    }
```

Comment:
The FrameworkBundle registers an exception handler, so we need to clean it up or PHPUnit complains.